rmf_rviz_plugin/src/ParseActionPlan.cpp

```diff
@@ -0,0 +1,216 @@
+#include "ParseActionPlan.hpp"
+#include <yaml-cpp/yaml.h>
+#include <iostream>
+
+namespace rmf_rviz_plugin {
+
+rmf_utils::optional<ActionPlan> parse_yaml_config(std::string path_name)
```

Comment:
since we are not worried of storing `path_name` anywhere, maybe pass by `const &`?

Reply:
:+1: